src/libslic3r/PrintConfig.cpp

```diff
@@ -665,6 +666,16 @@ void PrintConfigDef::init_fff_params()
     def->max = 300;
     def->set_default_value(new ConfigOptionInts{ 35 });
 
+    def = this->add("textured_cool_plate_temp", coInts);
+    def->label = L("Other layers");
+    def->tooltip = L("Bed temperature for layers except the initial one. "
+        "Value 0 means the filament does not support to print on the Textured Cool Plate");
```

Comment:
Just copy the tooltip from the other plates because "the filament does not support to print on the Textured Cool Plate" is pretty bad English, maybe have a native speaker review PRs where English is added to the UI.